genotype/store/trending.py

```
@@ -0,0 +1,62 @@
+"""Bla bla. What do you want here???"""
+from genotype.store.models import Analysis, Genotype, Sample
+
+
+def build_snp_dict(analysis_id):
+    """Building a dict of snps for a specific analysis."""
+
+    snp_dict = {}
+    try:
+        genotypes = Genotype.query.filter(Genotype.analysis_id == analysis_id).all()
```

could you rewrite this in a way that does not swallow all exceptions?